tests/common/fixtures/duthost_utils.py

```diff
@@ -440,17 +438,23 @@ def dut_qos_maps(rand_selected_dut):
     or an empty dict if failed to parse the output
     """
     maps = {}
+    dut = get_src_dst_asic_and_duts['src_dut']
     try:
+        if dut.is_multi_asic:
+            sonic_cfggen_cmd = "sonic-cfggen -n asic0 -d --var-json"
```

Comment:
@sanmalho-git - do we only need to build qos_map for 1 asic on multi-asic platforms?

Reply:
Assumption here is that all the asics have the same qos_map.

Reply:
thanks, it makes sense, qos_sai_base is also per host based.

Reply:
it would be better to use `enum_frontend_asic_index` in sonic-cfggen commands.

Reply:
enum_frontend_asic_index interferes with the parameterization of the tests for src/dst asic/dut index. So, can't include any of the enum fixtures